services/gundeck/src/Gundeck/Push/Websocket.hs

```diff
@@ -355,3 +357,6 @@ logPresence :: Presence -> Log.Msg -> Log.Msg
 logPresence p =
        Log.field "user"   (toByteString (userId p))
     ~~ Log.field "zconn"  (toByteString (connId p))
+
+timeout' :: Timeout -> (Request -> Request)
+timeout' = timeout . (`div` 1000000) . fromIntegral{- yields nanoseconds -}
```

Comment:
We should use the (#>) function provided by Data.Timeout here no?

`timeout' t = timeout $ (t #> Nanoseconds)` or something?

Reply:
Also we have waaaaay too many functions named timeout now haha; probably too much work to change this to `requestTimeout`; but maybe qualifying it as `Bilge.timeout` might help with reading these PRs